nnpdf_data/nnpdf_data/commondata/ATLAS_Z0J_8TEV/filter.py

```diff
@@ -0,0 +1,61 @@
+import yaml
```

Comment:
I'm noticing that the indentation in the Python files are not consistent. You could run a [pre-commit hooks](https://pre-commit.com/) to make sure that everything is properly formatted.